requirements.txt

```diff
@@ -26,6 +29,7 @@
 
 ## Dependencies
 
-grakn-protocol==2.0.0a9
+# IMPORTANT: Any changes to these dependencies should be copied to requirements_dev.txt.
```

Comment:
People kept forgetting to update `requirements_dev.txt`, so now I've left a glaring message telling them to do so.